src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -319,8 +327,7 @@ class TSPSchoolSyncer extends mix(Syncer).with(ClassImporter) {
 			this.stats.errors.push({
 				type: 'create-teacher',
 				entity: tspTeacher.lehrerUid,
-				message: `Lehrer "${tspTeacher.lehrerVorname} ${tspTeacher.lehrerNachname}"`
-					+ ' konnte nicht erstellt werden.',
+				message: `Lehrer "${tspTeacher.lehrerVorname} ${tspTeacher.lehrerNachname}"` + ' konnte nicht erstellt werden.',
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)